packages/ui/components/checkbox-group/src/LionCheckboxIndeterminate.js

```diff
@@ -197,16 +197,20 @@ export class LionCheckboxIndeterminate extends LionCheckbox {
   // eslint-disable-next-line class-methods-use-this
   _afterTemplate() {
     return html`
-      <div class="choice-field__nested-checkboxes">
+      <div class="choice-field__nested-checkboxes" role="list">
         <slot></slot>
       </div>
     `;
   }
 
   /**
+   * @param {Event} ev
    * @protected
    */
-  _onRequestToAddFormElement() {
+  _onRequestToAddFormElement(ev) {
+    if (!(/** @type {HTMLElement} */ (ev.target).hasAttribute('role'))) {
```

Comment:
are the [role=list] (shadow) and [role=lisitem] (probably light) in the same dom and does this work well in SR?

Reply:
They are in different doms, but this works in VoiceOver and NVDA

Reply:
Ok nice, as long as we don't have idrefs it's fine indeed :)